docs/setup/settings.asciidoc

```diff
@@ -296,14 +308,137 @@ This value must be a whole number greater than zero. *Default: `"1000"`*
+Instead, use the <<`rolling-file`, logging.appenders.rolling.file>> appender.
+ +
+experimental[] The number of most recent rotated log files to keep
+on disk. Older files are deleted during log rotation. The default value is 7. The `logging.rotate.keepFiles`
+option has to be in the range of 2 to 1024 files. *Default: `7`*
+
+`logging.rotate.pollingInterval`::
+deprecated:[7.13.0,This setting will be removed in Kibana 8.0.]
+Instead, use the <<`rolling-file`, logging.appenders.rolling.file>> appender.
+ +
+experimental[] The number of milliseconds for the polling strategy in case
+the <<logging-rotate-usePolling, `logging.rotate.usePolling`>> is enabled. `logging.rotate.usePolling` must be in the 5000 to 3600000 millisecond range. *Default: `10000`*
+
+[[logging-rotate-usePolling]] `logging.rotate.usePolling`::
+deprecated:[7.13.0,This setting will be removed in Kibana 8.0.]
+Instead, use the <<`rolling-file`, logging.appenders.rolling.file>> appender.
+ +
+experimental[] By default we try to understand the best way to monitoring
+the log file and warning about it. Please be aware there are some systems where watch api is not accurate. In those cases, in order to get the feature working,
+the `polling` method could be used enabling that option. *Default: `false`*
+
+`logging.silent`::
+deprecated:[7.13.0,This setting will be removed in Kibana 8.0.]
+ Instead, set the root logger `level` to `off`.
+ +
+Set the value of this setting to `true` to
+suppress all logging output. *Default: `false`*
+
+`logging.timezone`::
+deprecated:[7.13.0,This setting will be removed in Kibana 8.0.]
+Instead, use the {kibana-ref}/logging-service.html#pattern-layout[`pattern` appender layout] and specify a `%date` conversion pattern.
+ +
+Set to the canonical time zone ID
+(for example, `America/Los_Angeles`) to log events using that time zone.
+For possible values, refer to
+https://en.wikipedia.org/wiki/List_of_tz_database_time_zones[database time zones]. *Default: `UTC`*
+
+[[logging-verbose]] `logging.verbose` {ess-icon}::
+deprecated:[7.13.0,This setting will be removed in Kibana 8.0.]
+Instead, set the root logger `level` to `debug`.
+ +
+Set to `true` to log all events, including system usage information and all
+requests. *Default: `false`*
+
 NOTE: To reload the <<logging-settings, logging settings>>, send a SIGHUP signal to {kib}.
 
+// new logging configuration settings
 [[logging-root]] `logging.root`::
-The `root` logger has is a <<dedicated-loggers, dedicated logger>> and is pre-configured. The `root` logger logs at `info` level by default. If any other logging configuration is specified, `root` _must_ also be explicitly configured.
+The `root` logger has is a <<dedicated-loggers, dedicated logger>> and is pre-configured. The `root` logger logs at `info` level by default. If any other logger is configured, the `root` logger _must_ also be explicitly configured with an appropriate appender. 
```

Comment:
> If any other logger is configured, the `root` logger _must_ also be explicitly configured with an appropriate appender.

Is that true? I think this is still a valid config:

```yaml
logging:
  root:
    level: warn
  loggers:
    - name: elasticsearch.query
      level: debug
```

Reply:
um, you're right, I've just tested it.

Reply:
```suggestion
The `root` logger has is a <<dedicated-loggers, dedicated logger>> and is pre-configured. The `root` logger logs at `info` level by default. 
```

Reply:
Ah, now I remember what I was trying to say: "If any other logger is configured, the `root` logger _must_ also be explicitly defined/configured in `kibana.yml`". "with an appropriate appender" is superfluous.